test/models/chan.js

```diff
@@ -32,9 +32,7 @@ describe("Chan", function() {
 		};
 
 		var getUserNames = function(chan) {
-			return chan.users.map(function(u) {
-				return u.name;
-			});
+			return chan.users.map(u => u.nick);
```

Comment:
My bad about the map. But I would leave this discussion for another time (e.g. when we update eslint as I have a branch with extra rules), as existing code in repo already uses the same style.